### DISCUSSION TITLE
Simple 3D printed case for PCB V2

### DISCUSSION THREAD

**Discussion by janmittner** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#513] [Show and tell]
I have S100101 pump where you can place the unit inside the pump. However still better to be somehow protected. So I designed just very simple case which fits PCB V2. It is a tight fit, so no screws needed at all. Just use some tape around it to be sure.

https://www.thingiverse.com/thing:6030872

**Comment by visualapproach** [comment]
Nice 👍🏼
